6-anthos-install/acm-configs/config-management-east.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: configmanagement.gke.io/v1
```

Comment:
Some finding from the lint test we should fix
```
Checking for trailing whitespace
./6-anthos-install/README.md:1:# Deploying Bank of Anthos 
./6-anthos-install/README.md:92:We need to configure endpoint discovery for cross-cluster load balancing and communication. 
./6-anthos-install/README.md:102:3. In a similar manner, creates a secret that grants access to the Kube API Server for cluster 2 
./6-anthos-install/README.md:110:``` 
./6-anthos-install/README.md:144:With MCI, we need to select a cluster to be the configuration cluster. In this case, `gke-mci-us-east1-001`. Once MCI is enabled, we can setup MCI. This entails establishing namespace sameness between the clusters, deploying the application in the clusters as preferred (in `gke-boa-us-east1-001` and `gke-boa-us-east1-001` clusters), and deploying a load balancer by deploying MultiClusterIngress and MultiClusterService resources in the config cluster 
./6-anthos-install/README.md:152:2. Given that MCI will be used to loadbalance between the istio-gateways in east and west clusters, we need to create istio-system namespace in Ingress cluster to establish namespace sameness. 
./6-anthos-install/README.md:240:### Configure ACM 
./6-anthos-install/README.md:247:### Populate the CSR repos 
./6-anthos-install/README.md:257:cd ${HOME}/root-config-repo 
Error: Trailing whitespace found in the lines above.

Checking for missing newline at end of file
Error: No newline at end of file ./6-anthos-install/acm-configs/config-management-east.yaml
Error: No newline at end of file ./6-anthos-install/acm-configs/config-management-west.yaml
Error: No newline at end of file ./6-anthos-install/acm-configs/root-sync.yaml
Running shellcheck
Checking file headers
3 files have incorrect boilerplate headers:
6-anthos-install/acm-configs/config-management-east.yaml
6-anthos-install/acm-configs/config-management-west.yaml
6-anthos-install/acm-configs/root-sync.yaml
```